overhead_analysis/split_jobs.py

```diff
@@ -0,0 +1,40 @@
+import os 
+
+srcdir = "job_files"
+destdir = "single_jobs"
+
+if not os.path.exists(destdir):
+    os.mkdir(destdir)
+
+for fname in os.listdir(srcdir): 
```

Comment:
This approach seems slightly fragile compared to using stonewall in the job description, what do we gain from doing the splitting manually vs. that approach?

Reply:
I didn't know that it can specify a single job/config to run via fio's command line arguments. I'll try that and avoid using this manual split.